weave-js/src/components/Panel2/availablePanels.tsx

```diff
@@ -254,6 +255,12 @@ export function getPanelStacksForType(
       );
     }
 
+    if (opts.excludePanelPanel) {
+      handlerStacks = handlerStacks.filter(
+        hs => !PanelLib.getStackIdAndName(hs).id.endsWith('panel')
```

Comment:
are there any other panel ids that end with `panel`? perhaps you want to exclusively look for == panel?